motion_computation/test/MotionComputationTest.cpp

```diff
@@ -80,13 +80,26 @@ namespace object
         ASSERT_EQ(otValid, true);
 
         /*Test ExternalObjectList*/
-        cav_msgs::ExternalObjectList obj_list, test_list;
-        obj_list.objects.push_back(msg);
-        ASSERT_TRUE(obj_list.objects.size() > 0);
+        cav_msgs::ExternalObjectList obj;
```

Comment:
If you publish from your unit tested function you should validate the publication callback was actually triggered. 

At line 52 you create a lambda that can be leveraged for this

```
bool published_data = false;
object::MotionComputationWorker mcw([&](const cav_msgs::ExternalObjectList& obj_pub){ published_data =true });

....

//call motion prediction function
// assert that published_data  is true
```